cookies/operations.py

```diff
@@ -10,11 +10,52 @@
 from itertools import groupby
 from itertools import combinations
 
+import goat
+import unittest, mock, json, os, sys
+
 from cookies.exceptions import *
 import networkx as nx
 from collections import Counter
 logger = settings.LOGGER
 
+os.environ.setdefault('GOAT_WAIT_INTERVAL', '0.001')
+
+goat.GOAT_APP_TOKEN = 'd22bbda9b5b507dc6cd032d80d6a3d299fda10fe'
```

Comment:
Luckily this one wasn't active. But if it were, this would be a major security breach.